docs/src/site/apt/index.apt.vm

```diff
@@ -218,4 +211,10 @@ Notices
 </build>
 +------+
 
+ * Since version <<34>> <<<repositories>>> section has been moved to the <<<use-apache-snapshots>>> profile.
+
+   Apache snapshot repository is now optional, and is only added when the profile is activated.
+
+   It can be activated by setting the <<<apache.snapshots>>> property or activating the <<<use-apache-snapshots>>> profile directly.
```

Comment:
setting the property to what? I guess `true`? but maybe it's "active", "enabled". We should mentioned it whats proper values.

Reply:
simply use user property as `-Dsomething` set default value as `true`
I would like to left is as is to not write a long description ...